.flake8

```
@@ -0,0 +1,11 @@
+[flake8]
+select = ANN,B,B9,BLK,C,D,DAR,E,F,I,S,W
+ignore = E203,E501,W503
+max-line-length = 80
+max-complexity = 10
+application-import-names = esa_geo_utils,tests
```

This should be our package

```suggestion
application-import-names = sds_data_model,tests
```